control/control.go

```diff
@@ -12,6 +12,8 @@ type Controllable interface {
 	SetState(ctx context.Context, state []*Signal) error
 	// Position returns the current encoder count value
 	State(ctx context.Context) ([]float64, error)
+	// Type returns if the controllable is a base or a motor
+	GetType() string
```

Comment:
We should add this because a cotnrollable's interface only cares that `SetState` and `State` are implemented. If we ever need to check for logic within the drivers, not the controls package, we can do an `ok :=`controllable.(base.Base)` type assertion in go.